pxtblocks/plugins/renderer/pathObject.ts

```diff
@@ -48,8 +48,41 @@ export class PathObject extends Blockly.zelos.PathObject {
         super.updateSelected(enable);
     }
 
-    override addConnectionHighlight(connection: Blockly.RenderedConnection, connectionPath: string, offset: Blockly.utils.Coordinate, rtl: boolean): void {
-        super.addConnectionHighlight(connection, connectionPath, offset, rtl);
+    override addConnectionHighlight(connection: Blockly.RenderedConnection, connectionPath: string, offset: Blockly.utils.Coordinate, rtl: boolean): SVGElement {
```

Comment:
This is a bit unfortunate. Matt/Rob to discuss and see if there's something we can feed back to Blockly here.

Reply:
We looked at this and don't have a straightforward suggestion here.

Blockly added a return type and moved to proactively calling `addConnectionHighlight`, controlling visibility with CSS `display` and only removing the highlight in `dispose`.

By default this meant our override always showed the custom part.

So we changed our override to return a group so that both parts are controlled by the same CSS display change Blockly makes. But that requires a bit of copy/paste and access to the connectionHighlights map in super.

Suggestions for now this could be improved very welcome.